timelock-corruption-detection/src/main/conjure/timelock-history.yml

```diff
@@ -0,0 +1,52 @@
+types:
+  imports:
+    PaxosValue:
+      base-type: any
+      external:
+        java: com.palantir.paxos.PaxosValue
+    PaxosAcceptorState:
+      base-type: any
+      external:
+        java: com.palantir.paxos.PaxosAcceptorState
```

Comment:
I think for this one we should probably still have a separate layer, because this class is used internally within the Paxos implementation. (Unlike PaxosValue, this one isn't a part of TimeLock's internode API.)